tests/ballerina-tools-integration-test/src/test/java/org/ballerinalang/test/packaging/ImportModuleTestCase.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.ballerinalang.test.packaging;
+
+import org.apache.commons.io.FileUtils;
+import org.ballerinalang.test.BaseTest;
+import org.ballerinalang.test.context.BallerinaTestException;
+import org.ballerinalang.test.context.LogLeecher;
+import org.ballerinalang.test.utils.PackagingTestUtils;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.wso2.ballerinalang.compiler.util.ProjectDirConstants;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Map;
+
+/**
+ * Integration test cases for module imports.
+ *
+ * @since 0.985
+ */
+public class ImportModuleTestCase extends BaseTest {
+    private Path tempProjectDirectory;
+    private Path tempHomeDirectory;
+    private Map<String, String> envVariables;
+
+    @BeforeClass()
+    public void setUp() throws IOException {
+        tempHomeDirectory = Files.createTempDirectory("bal-test-integration-repo-hierarchy-home-");
+        tempProjectDirectory = Files.createTempDirectory("bal-test-integration-import-module-project-");
+
+        String projectPath = (new File("src/test/resources/importmodule")).getAbsolutePath();
```

Comment:
What is the naming convention used for folders multiple words ?  "import-module" ?